modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,98 @@
+= Portal
+
+`Unit ability`
+
+Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
+This can be an effective method to deploy reserve forces at the optimum time and place.
+
+Well-known examples include Necron portals and Eldar Webway portals.
+
+---
+
+[NOTE]
+====
+This ability lets you keep detachments outside of the play area at the start of the game, similar to Reserves.
+You can then deploy them later in the game.
+When they arrive, you disembark them into the play area through a unit -- or objective -- that has this ability.
```

Comment:
How will we apply this to an objective? Will there be a new type of objective especially for this, alongside the likes of Take and Hold?

Reply:
To be discussed on FB. This part is future-proofing for potential Eldar Webgates.

Reply:
Got it — now that I know Auto and Roll are mutually exclusive for a given army, it's making more sense. I'll make some tweaks so that these factors are more obvious.

Of course, for content in the experimental section we can cut some corners if you just want to get it out there — but I've defaulted to giving this much the same attention as I've given to the 'official' content that forms the bulk of this PR, since we're not rushing this through anyway.